### ISSUE TITLE
Does not work with 64bit system

### ISSUE THREAD

**Issue by welksonramos** [rxi/lovedos#12]
:(

**Comment by cooljith91112** [comment]
Of course, _lovedos_ will never work on a modern system it need a DOS machine. That's why its mentioned on the description as **A framework for making 2D DOS games in Lua**.
Use [DOS BOX](https://www.dosbox.com/) to run games build on _lovedos_.
